flow/ProtocolVersion.h

```diff
@@ -135,3 +135,6 @@ constexpr ProtocolVersion currentProtocolVersion(0x0FDB00B063010001LL);
 // This assert is intended to help prevent incrementing the leftmost digits accidentally. It will probably need to
 // change when we reach version 10.
 static_assert(currentProtocolVersion.version() < 0x0FDB00B100000000LL, "Unexpected protocol version");
+
+// Downgrades are only supported for one minor version
+constexpr ProtocolVersion minInvalidProtocolVersion(0x0FDB00B071000000LL);
```

Comment:
does this mean when we make release 7.1, the downgrade test will fail?

Reply:
In 7.0, `minInvalidProtocolVersion` will be set to `0x0FDB00B072000000LL` and in `7.1`, the `to_6.3.5` downgrade test will be replaced by a `to_7.0.0` downgrade test. We always will only test downgrades for one minor version.